ibm/service/satellite/resource_ibm_satellite_cluster.go

```diff
@@ -98,6 +98,11 @@ func ResourceIBMSatelliteCluster() *schema.Resource {
 				Computed:    true,
 				Description: "The lifecycle state of the cluster.",
 			},
+			"infrastructure_topology": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "String value for single node cluster option. Available options: single-replica, highly-available (default: highly-available)",
```

Comment:
Add validateFunc to allow single-replica and highly-available only as the values